Bigtable/src/ChunkFormatter.php

```diff
@@ -195,7 +195,11 @@ public function readAll()
         //   are to be discarded.
 
         foreach ($this->stream as $readRowsResponse) {
-            $this->prevRowKey = $readRowsResponse->getLastScannedRowKey();
+            if ($lastScannedRowKey = $readRowsResponse->getLastScannedRowKey()) {
+                // If the Response contains a "last_scanned_row_key", it's possible that there was
+                // a chunkless drop
```

Comment:
I think we can remove the comment because "chunkless drop" is not something server sends back. Server will just send back a response with lastScannedRowKey in some cases.